core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly

```diff
@@ -63,6 +63,14 @@ THE SOFTWARE.
         lazy="descriptor,it,${capture}">
         <l:ajax>
           <j:set var="instance" value="${current.descriptor==descriptor ? current : null}" />
+          <j:set var="help" value="${descriptor.helpFile}"/>
+          <j:if test="${help != null}">
+            <div class="help-sibling" style="width:100%">
+              ${%Overview}
+              <f:helpLink url="${help}" featureName="${descriptor.displayName}"/>
```

Comment:
This improves the spacing and contrast, top one is after, bottom one is before.

<img width="603" alt="image" src="https://user-images.githubusercontent.com/21194782/160693660-28ca78f2-65f8-4ced-a580-df405f4bcd8e.png">



```suggestion
              <p class="jenkins-form-label">${%Overview}
                <f:helpLink url="${help}" featureName="${descriptor.displayName}"/>
              </p>
```

Reply:
(trying something else)